### PR TITLE
Remove version downgrades, bump msrv to 1.46.0

### LINKED ISSUE #347: Revert #340
The bitflags version cannot be locked as discussed after the merge of #340 

Thank you @Imberflur for raising awareness to this.

### PR DESCRIPTION
**Pull request by torkleyy**
* [x] I've included my change in `CHANGELOG.md`

Fixes #347

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -24,7 +24,7 @@ integer128 = []
 
 [dependencies]
 base64 = "0.13"
-bitflags = "=1.2.1"
+bitflags = "1.0.4"
```
Comment (MomoLangenstein) [inline Cargo.toml:27]: What’s special about V1.0.4?

**Review by MomoLangenstein — APPROVED** [review]

File: Cargo.toml (hunk 1 above)
Comment (torkleyy) [reply Cargo.toml:27]: oh that was just the version we had required initially (I just did git revert)

**Review by MomoLangenstein — CHANGES_REQUESTED** [review]
Could you also resolve the `// TODO: use fXX::EPSILON when minimum supported rust version is 1.43` comments in this MSRV bump?

**Comment by enfipy** [comment]
We are also blocked by this. Can we push this forward? ❤️

**Comment by MomoLangenstein** [comment]
> We are also blocked by this. May we push this forward? ❤️

As @torkleyy seems to be busy at the moment, I'll update the PR and get it merged today.

**Comment by MomoLangenstein** [comment]
Superseded by #361

**Pull request closed without merging**